spring-boot-project/spring-boot-actuator-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -116,6 +116,12 @@
       "description": "Whether to enable database health check.",
       "defaultValue": true
     },
+    {
+      "name": "management.health.db.ignore-routing-datasources",
+      "type": "java.lang.Boolean",
+      "description": "Whether to ignore the creation of health indicators for AbstractRoutingDatasource s",
+      "defaultValue": false
+    },
```

Comment:
If we switch to `@ConfigurationProperties` as suggested above, this change will no longer be necessary.

Reply:
Removed this.